src/ConnectedRouter.ts

```diff
@@ -95,15 +95,19 @@ const createConnectedRouter = (structure: Structure): React.FC<ConnectedRouterPr
         // @ts-ignore
         unpatchRouter = patchRouter(Router, store)
         Router.events.on('routeChangeStart', trackRouteStart)
-        Router.events.on('routeChangeError', onRouteChangeFinish)
+        Router.events.on('routeChangeError', trackRouteComplete)
```

Comment:
because the arguments for this event handler are `(err, url)` and not `(url)`

Reply:
Good catch!